src/Backend/app.py

```diff
@@ -41,7 +41,7 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
 import json
-from src.Backend.utils import *
+from utils import *
```

Comment:
Please revert it to the previous version

Reply:
Reverted